index.js

```diff
@@ -63,6 +65,31 @@ class Timber {
 		console.error(...args);
 	}
 
+	time(name) {
+		if (name) {
```

Comment:
I don’t think it should silently handle no name. Instead it should throw.

Reply:
I just checked `console.time`, and it nothing is passed, it assigns the name to `"default"`.

I think we should actually copy that, meaning:

```js
logger.time();
logger.timeEnd(); // logs "default: 1234ms"
```